tspng/test_implantation.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import pytest
+
+from io import BytesIO
+from tspng.implantation import implant,implant_from_file
+from tspng.extraction import extract
+
+def test_implant():
+    '''
+    Tests the dictionary keys from an example TSPNG file path.
+    '''
+    #test file
+    implant('tests/assets/coco_data.json','tests/assets/empty.png')
+    test_data = extract('tests/assets/empty.png_MOD')
```

Comment:
Once the `.ts.png` file extension is applied, this needs to be updated and then the `tests/assets/empty.ts.png` file will need to be deleted after extracting the data but before the assertion so if any of the assertions that follow do not cause the file to linger.